frontends/mit-learn/src/pages/HomePage/HeroSearch.tsx

```diff
@@ -116,35 +112,47 @@ const ControlsContainer = styled.div(({ theme }) => ({
     },
   },
 }))
+
+const BrowseByTopicContainer = styled.div(({ theme }) => ({
+  marginTop: "16px",
+  marginBottom: "24px",
+  [theme.breakpoints.down("sm")]: {
+    marginTop: "0",
+  },
+}))
+
+const BrowseByTopicText = styled(Typography)(({ theme }) => ({
+  color: theme.custom.colors.silverGrayDark,
+  ...theme.typography.body2,
+}))
+
+const TopicLink = styled(Link)({
+  textDecoration: "underline",
+})
+
 const LinksContainer = styled.div(({ theme }) => ({
```

Comment:
Could you try removing this? I'm pretty sure `LinksContainer` can be replaced with an unstyled `div`.
- we aren't changing the flex-direction anymore at different breakpoints
- the direction is always column, so `width` isn't doing anything
    - so the container size is just the content size
        - so `justifyContent` isn't doing anything